features/multiple-memory-access-instructions/README.md

```diff
@@ -0,0 +1,11 @@
+# General design
+
+**Feature State: Mostly Stable**  
+**FEAT Bit: 3**
+
+* The memory instructions, including but not limited to `store`, `load`, `push`, and `pop`, _must_ accept memory operands as specified by the `MO1` and `MO2` extensions.
+* The `pop` instruction _must_ be treated as an _illegal_ instruction when the RMI byte follows the pattern `xxx 00x 01`.
```

Comment:
Please specify what this is preventing for clarity.

Reply:
Does the latest commit explain it well enough?

Reply:
Does this means it's legal to use a memory location as a stack pointer? That would mean three memory accesses, right?

Reply:
Yes it would. I think we should probably specify that it would only be usable as a stack pointer if they also have the arbitrary stack pointer extension?

Reply:
Even then it's kinda weird IMO. But yeah, asp should probably be a basic requirement for this.

Reply:
I added it in the feature